python_src/src/lamp_py/postgres/postgres_utils.py

```diff
@@ -342,8 +345,15 @@ def seed_metadata(self) -> None:
         process_logger.log_start()
         try:
             seed_file = os.path.join(
-                HERE, "..", "tests", "july_17_filepaths.json"
+                HERE,
+                "..",
+                "..",
+                "..",
+                "tests",
+                "test_files",
+                "july_17_filepaths.json",
             )
+            seed_file = os.path.abspath(seed_file)
```

Comment:
had to update the relative path to find the seed info here. i think my preference is to not do this as a command line argument for this since we then have to include this json file in our docker images if we want to run it in docker compose for local testing.

i have two suggestions for how we achieve this in the future (and we have [a ticket](https://app.asana.com/0/0/1204188000578034/f) to do so). 

1. write a script that would connect to our local rds spun up by docker compose and insert it
2. use volumes to add the file to the container when spawning and pass that file in as a command line argument

one seems easier imho.